### PR TITLE
Foreman 3.4 public key

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

**Review by Odilhao — APPROVED** [review]

File: security.md [hunk 1]
```diff
@@ -1074,6 +1074,15 @@ Key management is changing at the time of writing to cycle nightly keys every tw
         <td style='white-space:nowrap'></td>
         <td></td>
       </tr>
+      <tr>
+        <td><a href="{{ site.baseurl }}/static/keys/2A6F745AA62DE481567D3AF639D12077F120E13E.pub">2A6F745AA62DE481567D3AF639D12077F120E13E</a></td>                                                               
+        <td>FIXME: fingerprint</td>
```
Comment (ekohl) [inline security.md]: Please fix this

**Review by ekohl — CHANGES_REQUESTED** [review]
I tried to replicate this by importing the GPG secret, but the data in our shared storage looks wrong:
```console
$ VERSION=3.4 ./import_gpg_private 
✔ Copied theforeman/releases/foreman/3.4-gpg to clipboard. Will clear in 45 seconds.
gpg: keybox '/home/ekohl/dev/theforeman-rel-eng/releases/foreman/3.4/gnupg/pubring.kbx' created

Error: empty secret
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
```

There is a gpg passphrase but I don't see the actual key.

@Odilhao next time you review a PR like this, please perform the same steps. This is crucial.

File: security.md (hunk 1 above)
Comment (pcreech) [reply security.md]: *facepalm* whoops, will do.

**Comment by pcreech** [comment]
@ekohl as far as the key, I'm pretty sure I followed all steps for key generation, but i'll go back and check to see if I can find the breakdown.

**Comment by ekohl** [comment]
> @ekohl as far as the key, I'm pretty sure I followed all steps for key generation, but i'll go back and check to see if I can find the breakdown.

I tried to look at the git log, but I'm not sure what went wrong. Perhaps you mistyped a passphrase or something and it stored garbage?

**Comment by Odilhao** [comment]
```
gpg: keybox 'theforeman-rel-eng/releases/foreman/3.4/gnupg/pubring.kbx' created
gpg: /home/osousa/code/sat/theforeman-rel-eng/releases/foreman/3.4/gnupg/trustdb.gpg: trustdb created
gpg: key 39D12077F120E13E: public key "Foreman Automatic Signing Key (3.4) <packages@theforeman.org>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg:       secret keys read: 1
gpg:   secret keys imported: 1
```

File: security.md (hunk 1 above)
Comment (pcreech) [reply security.md]: should be fixed

**Comment by pcreech** [comment]
@ekohl the key should be good to go.

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**